docs/sources/best-practices/current-best-practices.md

```diff
@@ -105,7 +105,7 @@ It's also worth noting that the batching nature of the Loki push API can lead to
 
 ## 7. Use `chunk_target_size`
 
-This was added earlier this year when we [released v1.3.0 of Loki](https://grafana.com/blog/2020/01/22/loki-1.3.0-released/), and we've been experimenting with it for several months. We have `chunk_target_size: 1536000` in all our environments now. This instructs Loki to try to fill all chunks to a target _compressed_ size of 1.5MB. These larger chunks are more efficient for Loki to process.
+This was added earlier this year when we released [Loki v1.3.0](https://grafana.com/blog/2020/01/22/loki-1.3.0-released/), and we've been experimenting with it for several months. We have `chunk_target_size: 1536000` in all our environments now. This instructs Loki to try to fill all chunks to a target _compressed_ size of 1.5MB. These larger chunks are more efficient for Loki to process.
```

Comment:
This was added earlier in the Loki v1.3.0 release (basically make the sentence active voice, and remove passive voice).